### ISSUE TITLE
Logger 

### ISSUE THREAD

**Issue by skynigh** [TomRichter/ppl-common-extensions#3]
i assume some way to track issue or have a output of some kind that writes to a file

**Comment by TomRichter** [comment]
Yep, just straight forward logging.  PULSAR already captures any `Console.WriteLine` or standard Unity logging method into its own [`output_log.txt` file](http://www.pulsarthegame.com/sending-an-output-log-file/), so we just need to take advantage of that while disambiguating which plugin is calling the log.

A wrapper around [Unity's logging](https://docs.unity3d.com/2017.4/Documentation/ScriptReference/Logger.html) is probably sufficient.  It already supports multiple log levels, and we can "tag" lines with the current plugin name.  If we need something fancier in the future, [NLog](https://github.com/NLog/NLog/wiki/Tutorial) writing to stdout will be more than enough.
